src/lib/y2firewall/clients/auto.rb

```diff
@@ -84,13 +126,23 @@ def modified?
 
     private
 
+      # Depending on the profile it activate or not the firewalld service
```

Comment:
*it activates*. On the other hand, you could say something like *it activates or deactivates* (because "not activate" != "deactivate").